ei-distributed-transactions/README.md

```diff
@@ -0,0 +1,63 @@
+##EI Distributed Transactions setup
```

Comment:
The heading doesn't render properly in the readme page. Need to add a space after ## symbol.

Reply:
Fixed with https://github.com/wso2/platform-scenarios/pull/11/commits/8cd0126e6cd659415fdf1de129e72841e1006d26